packages/examples/src/react-focus/FocusZone/examples/FocusZone.Photos.Example.tsx

```diff
@@ -46,9 +45,24 @@ interface IPhoto {
   height: number;
 }
 
+const getItems = (): IPhoto[] => {
+  const items: IPhoto[] = [];
+  for (let i = 0; i < MAX_COUNT; i++) {
+    const randomWidth = 50 + Math.floor(Math.random() * 150);
+    items.push({
+      id: useId('photo'),
```

Comment:
In this case you should actually still do `getId` since wrapping in `useConst` guarantees it will only be called once. Otherwise you're conditionally doing a hook call (only calling it on first render), which isn't good.

Also this might be just preference, but I generally prefer leaving secondary things like this below the example to make it easier for someone opening the example code on the website to quickly understand the main point.